internal/pkg/api/server/templates/service/inputs_test.go

```diff
@@ -0,0 +1,339 @@
+package service
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	. "github.com/keboola/keboola-as-code/internal/pkg/api/server/templates/gen/templates"
+	"github.com/keboola/keboola-as-code/internal/pkg/template"
+	"github.com/keboola/keboola-as-code/internal/pkg/template/input"
+)
+
+type validateCase struct {
+	name string
+	// Input
+	groups  template.StepsGroups
+	payload []*StepPayload
+	// Expected
+	result *ValidationResult
+	values template.InputsValues
+	err    string
+}
+
+func TestValidateInputs(t *testing.T) {
+	t.Parallel()
+
+	cases := []validateCase{
+		// One group, with one step, with one input
+		{
+			name: "simple",
+			groups: template.StepsGroups{
+				{
```

Comment:
Pridal som aj dalsie unit testy, kedze ten `/validate` endpoint, je uz celkom stabilny, a je uz stabilne dohodnute, ako to presne funguje.